app/src/main/java/com/example/habitsmasher/PublicPrivateButtons.java

```diff
@@ -0,0 +1,131 @@
+package com.example.habitsmasher;
+
+import static android.content.ContentValues.TAG;
+
+import android.nfc.Tag;
+import android.util.Log;
+import android.view.View;
+import android.widget.Button;
+import android.widget.CheckBox;
+
+import com.google.android.material.tabs.TabLayout;
+
+public class PublicPrivateButtons {
```

Comment:
Made a class for the buttons, in the future I'm gonna refactor the DaysTracker to also have all of button logic included.

Reply:
Great job!

Reply:
I like this idea!